common/viewer.h

```diff
@@ -179,6 +179,7 @@ namespace rs2
         bool show_help_screen = false;
         bool occlusion_invalidation = true;
         bool glsl_available = false;
+        bool expended_notification_on = false;
```

Comment:
expend -> expand

Reply:
call it what it is: `modal_notification` and if there's confusion: `// a notification which was expanded`

Reply:
Done